### PR TITLE
feat : already created positions in department are now editable

### PR DESCRIPTION
**Pull request by Salg-0**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "private": true,
   "homepage": "https://tsb.tecnico.ulisboa.pt/",
   "scripts": {
-    "start": "GENERATE_SOURCEMAP=false PORT=3005 react-scripts start",
-    "build": "GENERATE_SOURCEMAP=false react-scripts build",
+    "start": "set \"GENERATE_SOURCEMAP=false\" PORT=5000 && react-scripts start",
+    "build": "set \"GENERATE_SOURCEMAP=false\" && react-scripts build",
```
Comment (dinis-rodrigues) [inline package.json]: Let's not do this.
I would prefer to, instead, do this cleaner solution: [https://stackoverflow.com/a/42232944/12771385](https://stackoverflow.com/a/42232944/12771385)
And update the `template.env`

File: src/Pages/Admin/DepartmentManagement/DepartmentModal.tsx [hunk 2]
```diff
@@ -236,10 +237,34 @@ const DepartmentModal = ({
...
+                      departmentInfo,
+                      setDepartmentInfo
+                    );
+                  }}
+                  // on finish change editPosition(
+                  //       position,
+                  //       idx,
+                  //       departmentInfo,
+                  //       setDepartmentInfo
+                  //     );
```
Comment (dinis-rodrigues) [inline src/Pages/Admin/DepartmentManagement/DepartmentModal.tsx]: Any reason for this, or can we remove the comment?

File: package.json (hunk 1 above)
Comment (dinis-rodrigues) [reply package.json]: To be clear on this, revert these changes

File: src/Pages/Admin/DepartmentManagement/DepartmentModal.tsx (hunk 2 above)
Comment (Salg-0) [reply src/Pages/Admin/DepartmentManagement/DepartmentModal.tsx]: forgoten comment, will just remove them

File: package.json (hunk 1 above)
Comment (Salg-0) [reply package.json]: both port and GENERATE_SOURCEMAP=false are now in .env

**Review by dinis-rodrigues — APPROVED** [review]
LGTM 🚀

**Pull request merged**